docs/AboutUs.md

```diff
@@ -7,53 +7,45 @@ We are a team based in the [School of Computing, National University of Singapor
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
+## CoLAB Project team
 
-### John Doe
+### Fang Samuel
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/samuelfangjw.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/samuelfangjw)]
+[[portfolio](team/samuelfangjw.md)]
 
-* Role: Project Advisor
-
-### Jane Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Team Lead
+* Role: Deliverables and deadlines
```

Comment:
perhaps we can change the phrasing of the roles for all of us to `In charge of ...`. e.g. In charge of documentation, In charge of Integration. Currently the role sections sound a bit weird for all of us.

Reply:
yeah i think its weird yesterday too, but not sure thanks for the comment

Reply:
In that case would it be 

In charge of: Deliverables and deadlines
vs
Role: Deliverables and deadlines
vs
Role: In charge of Deliverables and Deadlines